update/provisioner.go

```diff
@@ -22,7 +22,6 @@ const (
 	elevatedPath          = "C:/Windows/Temp/packer-windows-update-elevated.ps1"
 	elevatedCommand       = "PowerShell -ExecutionPolicy Bypass -OutputFormat Text -File C:/Windows/Temp/packer-windows-update-elevated.ps1"
 	windowsUpdatePath     = "C:/Windows/Temp/packer-windows-update.ps1"
-	windowsUpdateCommand  = "PowerShell -ExecutionPolicy Bypass -OutputFormat Text -File C:/Windows/Temp/packer-windows-update.ps1"
```

Comment:
can you please retain this variable? but add an ` %d` at the end and use `fmt.Sprintf` when its needed? the reasoning is to have a central place to glimpse all the used commands.